docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -123,18 +154,27 @@ Standard TensorFlow\* operations:
 
 | Operation Name in TensorFlow\* | Limitations|
 | :----------| :----------|
+| Abs | No |
 | Acosh | No |
 | Add | No |
 | AddV2 | No |
 | AddN | No |
+| All | No |
 | ArgMax | No |
 | ArgMin | No |
 | Asinh | No |
+| Assert | No |
+| Assign | No |
+| AssignSub | No |
```

Comment:
@jane-intel, could you please comment here? What exact limitations do we have here?

Reply:
After offline discussion with @lazarevevgeny it was decided to write following limitation here:  Not needed for inference.